public/testem/decycle.js

```diff
@@ -49,7 +49,7 @@ function decycle(object, maxDepth) {
   maxDepth = maxDepth || 5;
   return (function derez(value, path, depth) {
     if (depth > maxDepth) {
-      return 'Max depth.';
+      return `Decycle max depth reached at ${maxDepth}. You may override the default by setting "client_decycle_depth" in your testem config.`;
```

Comment:
Not all browser testem supports, support string templates. Would you mind using string concatenation?